pyart/retrieve/qpe.py

```diff
@@ -1,18 +1,644 @@
-#! /usr/bin/env python
```

Comment:
This module is not used in Py-ART and can everything in the original module can be removed.